apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -126,6 +140,7 @@ export KUMA_PROTOCOL ?= "https://"
 export KUMA_SECURE_HSTS_SECONDS ?= 0
 export KUMA_SERVE_LEGACY ?= True
 export KUMA_SESSION_COOKIE_SECURE ?= True
+export KUMA_SSR_TIMEOUT ?= 10
```

Comment:
I think you had mentioned changing this to 5. Maybe you changed your mind. I don't think it matters much at all...

Reply:
Oh, good catch, yes!